python/rikai/types/geometry.py

```diff
@@ -60,67 +63,114 @@ def to_numpy(self) -> np.ndarray:
 
 
 class Box2d(ToNumpy):
-    """2-D Bounding Box.
+    """2-D Bounding Box, defined by ``(xmin, ymin, xmax, ymax)``
 
     Attributes
     ----------
-    x : float
-        X-coordinate of the center point of the box
-    y : float
-        Y-coordinate of the center point of the box
-    width : float
-        The width of the box
-    height : float
-        The height of the box
+    xmin : float
+        X-coordinate of the top-left point of the box.
+    ymin : float
```

Comment:
the terminology is a bit confusing because top-left should be `ymax` no?

Reply:
the top-left point is `(0,0)`

Reply:
ah ok. you think most users will already know the coordinate system or would they need a docstring reminder?

Reply:
while i know these are 3 most common coordinate systems, i am not sure the function is good tho.   open for suggestion.

Reply:
no i meant just add a note in the docstring above saying something like 'note: this uses graphics coordinate system where the top-left is (0,0)'